release-notes.txt

```diff
@@ -1,3 +1,729 @@
+* dist/tools/suit: encrypt an existing key (#18499)
+* doccheck: remove exclude_patterns not needed (#18597)
+* makefiles/arch/avr8.inc.mk: Fix compilation with GCC 12.2.0 (#18533)
+* makefiles/boards/stm32.inc.mk: improve stm32flash support (#18452)
+* makefiles: Select suitable terminal when stdio_rtt is used (#18528)
+* pseudomodules: document gnrc_netif_cmd_lora as deprecated (#18706)
+
+Kconfig
+-------
+
++ cpu/esp32: add ESP32-S2 support in Kconfig (#18504)
++ cpu/esp32: add ESP32-S3 support in Kconfig (#18411)
++ cpu/esp32: split Kconfig into ESP32x common and ESP32 specific parts (#18320)
++ drivers: model IEEE 802.15.4 devices in Kconfig (#17789)
++ drivers: model kconfig for Ethernet drivers (#17739)
++ Kconfig/ieee802154: fix typo in HAL selector and add missing radios (#18430)
+* .murdock: disable hash checks of kconfig/make (#18423)
+* cpu/esp_{common,32,8266}: improve Kconfig submenu structure (#18314)
+* drivers/kw2xrd/Kconfig: fix kconfig model (#18497)
+* Fix kconfig models breaking nightlies (#18718)
+
+Examples
+--------
+
++ examples/gcoap_dtls: Add coap-client example (#18660)
+* examples/filesystem: drop manual mounts (#18495)
+* examples: Move in RIOT/applications (#18602)
+
+Testing
+-------
+
++ .murdock.yml: Add preview link to documentation to comment (#18677)
++ release-tests: add capability to run on PR comment (#18713)
++ Revert "release-tests: add capability to run on PR comment" (#18736)
++ tests/external_boards: add esp32c3-ci board for optional modules (#18404)
++ tests/external_boards: add esp32s2-ci board for optional modules (#18507)
++ tests/external_boards: add esp32s3-ci board for optional modules (#18422)
++ tests/gcoap_fileserver: add integration test for GCoAP fileserver (#18199)
+* CI: improve makefile broken handling (#18595)
+* ci: teach github-labeler about .murdock.yml (#18666)
+* gh-actions: move actions/checkout to main (#18730)
+* murdock: honour RUN_TESTS setting (#18546)
+* murdock: make available boards instance dependent (#18659)
+* riotctrl_shell.cord_ep: remove regif parameter (#18716)
+* tests/ieee802154_*: fix tx_done handler on ACK_TIMEOUT and improve
+  tests (#18382)
+* tests/periph_gpio_ll: drop core dump merged by accident (#18541)
+* tests/periph_timer_short_relative_set: clarify outcome (#18513)
+* tests/pkg_lvgl_touch: randomize button position on click (#18481)
+* tests/pkg_tinyvcdiff: Initialize mtd->write_size (#18529)
+* tests/thread_priority_inversion: blacklist esp32 (#18535)
+* tests/ztimer_periodic: improve output and relax on native (#18491)
+* tests/{periph_gpio_ll,pkg_fff}: change CPU_FAM to MCU for ESP32x SoCs (#18356)
+
+API Changes
+-----------
+
++ drivers/kw2xrf: add support for IEEE 802.15.4 Radio HAL (#18383)
++ sys/shell: make cmds submodules and add KConfig modeling (#18355)
+* cpu/sam0_common: adc: drop pin from adc_conf_chan_t (#18658)
+* hashes_cmac: rename to hashes_aes128_cmac (#18539)
+* nanocoap_sock: consitfy remote (#18588)
```

Comment:
```suggestion
* nanocoap_sock: consitfy remote (#18588)
* gnrc_netreg: Use locks around netreg;  calls to gnrc_netreg_lookup and
  gnrc_netreg_getnext now need to be guarded (#18740)
```

This is an API change, and a subtle one at that (because the compiler won't complain).

Reply:
Also, I think the word must be `constify` ;-).